tests/cache-test/src/testCache.c

```diff
@@ -0,0 +1,64 @@
+#include <assert.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "cache.h"
+#define TESTSIZE 1
+
+void printValue(int *val)
+{
+    if (val)
+        printf("%d\n", *val);
+    else
+        printf("NULL\n");
+}
+
+void split(char **arr, char *str, const char *del)
+{
+    char *s = strtok(str, del);
+
+    while (s != NULL) {
+        *arr++ = s;
+        s = strtok(NULL, del);
+    }
+}
+
+int main(__attribute__((unused)) int argc, char *argv[])
```

Comment:
Replace `_attribute__((unused))` with `UNUSED` since `common.h` already defines.

Reply:
You shall check `argc` in advance.